pygrocy/grocy_api_client.py

```diff
@@ -168,31 +168,14 @@ def next_estimated_execution_time(self) -> datetime:
         return self._next_estimated_execution_time
 
 
-class CurrentVolatilStockResponse(object):
-    def __init__(self, parsed_json):
-        self._expiring_products = [ProductData(product) for product in parsed_json.get('expiring_products')]
-        self._expired_products = [ProductData(product) for product in parsed_json.get('expired_products')]
-        self._missing_products = [ProductData(product) for product in parsed_json.get('missing_products')]
-
-    @property
-    def expiring_products(self) -> List[ProductData]:
-        return self._expiring_products
-
-    @property
-    def expired_products(self) -> List[ProductData]:
-        return self._expired_products
-
-    @property
-    def missing_products(self) -> List[ProductData]:
-        return self._missing_products
-
 
 class CurrentStockResponse(object):
     def __init__(self, parsed_json):
         self._product_id = parse_int(parsed_json.get('product_id'))
         self._amount = parse_float(parsed_json.get('amount'))
         self._best_before_date = parse_date(parsed_json.get('best_before_date'))
-
+        self._amount_opened = parse_int(parsed_json.get('amount_opened'))
```

Comment:
Is this really an int or should this be a float?

Reply:
I don't know. Amount is a float. But I can not find a way to "open" 1.x
And documentation is not up to date :/